sdk/formrecognizer/azure-ai-formrecognizer/tests/test_dmac_mgmt.py

```diff
@@ -106,7 +105,7 @@ def test_mgmt_model(self, client, formrecognizer_storage_container_sas_url, **kw
             compare_bodies=False, excluded_headers="Authorization,Content-Length,x-ms-client-request-id,x-ms-request-id"
         )
         
-        poller = client.begin_build_model(formrecognizer_storage_container_sas_url, description="mgmt model")
+        poller = client.begin_build_model(formrecognizer_storage_container_sas_url, "template", description="mgmt model")
```

Comment:
do we not have an async version of this test?

Reply:
Hmm I thought I'd checked for this. Just added it!